packages/ts/generator-typescript-plugin-backbone/src/EndpointMethodRequestBodyProcessor.ts

```diff
@@ -18,54 +18,87 @@ export type EndpointMethodRequestBody = ReadonlyDeep<OpenAPIV3.RequestBodyObject
 export type EndpointMethodRequestBodyProcessingResult = Readonly<{
   parameters: readonly ParameterDeclaration[];
   packedParameters?: ObjectLiteralExpression;
+  initParam: ts.Identifier;
 }>;
 
 export default class EndpointMethodRequestBodyProcessor {
+  readonly #defaultInitParamName = 'init';
+
   readonly #dependencies: DependencyManager;
   readonly #owner: Plugin;
   readonly #requestBody?: EndpointMethodRequestBody;
+  readonly #endpointRequestInitIdentifier: ts.Identifier;
 
   public constructor(
     requestBody: ReadonlyDeep<OpenAPIV3.ReferenceObject | OpenAPIV3.RequestBodyObject> | undefined,
     dependencies: DependencyManager,
     owner: Plugin,
+    endpointRequestInitIdentifier: ts.Identifier,
```

Comment:
I suggest this name to have `Type` part somewhere. For the first time, I was confused it with the parameter name. 
```suggestion
    endpointRequestInitTypeIdentifier: ts.Identifier,
```
Or we could simplify here as well:
```suggestion
    initTypeIdentifier: ts.Identifier,
```